### PR TITLE
update the hotfix script for tolerating null sources on projected volumes

### PR DESCRIPTION
**Pull request by zapman449**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
update the hotfix script for tolerating null sources on projected volumes

#### Which issue(s) this PR fixes:
hot fix build for #1497 

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

### CODE REVIEW

File: scripts/apply-hotfixes.sh [hunk 1]
```diff
@@ -74,4 +74,17 @@ else
     exit 1
 fi;
 
+# Patching commits for Tolerating Null Sources on Projected Volumes
+# UPDATE: OpenAPI generator v4.3.0 has the context manager as a functionality. Cherry-picking just the tests for completeness.
```
Comment (roycaihw) [inline scripts/apply-hotfixes.sh:78]: Thanks! Could you remove this line and add "TODO: remove this patch when we release v20 clients"? Otherwise LGTM

Comment (zapman449) [reply scripts/apply-hotfixes.sh]: done.

**Comment by roycaihw** [comment]
/lgtm
/approve

Thanks!

**Pull request merged**